script.module.inputstreamhelper/addon.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+''' This is the actual InputStream Helper plugin entry point '''
+
+from __future__ import absolute_import, division, unicode_literals
+import sys
+from inputstreamhelper import ADDON, Helper, log
+
+
+def run(params):
```

Comment:
Perfect, thanks

Reply:
Just a recommendation for the future: the add-on entrypoint is the only file which is not compiled to bytecode. For that reason is better to keep this file to a minimum and move the majority of the code to a module you can import. That way you can speed up a bit the loading time:

```python
import sys
import myaddon

myaddon.run(sys.argv)
```

Reply:
Okay, this will be fixed in a next release.